apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -109,9 +110,12 @@ const PromptCustomization: React.FunctionComponent = () => {
         )}
       </div>
       <div className={styles.footerButtonContainer}>
-        <button type="button" disabled={allFieldsDisabled} onClick={onUpdate}>
-          Update
-        </button>
+        <Button
+          text="Update"
```

Comment:
Add 'edit' icon here as well? I see it in mock up.

Reply:
Done!